### DISCUSSION TITLE
Deleting a Sandbox without sign in.

### DISCUSSION THREAD

**Discussion by StevenRoy52** [codesandbox/codesandbox-client#5573] [Q&A] [answered]
Hello,

I created a sandbox without signing in and saved the template. I would like to delete the template but it does not show the "Delete sandbox" in sandbox info. It always creates a fork when I make changes to the template which I am able to delete by going to sandbox info but the original template remains unchanged. Please advise on how to delete the original template.

Thanks,
Steven

**Comment by andrasbacsai** [comment, marked as answer]
Hey, 

For saving a sandbox as a template, you need to be logged in. Otherwise, it does not allow to do so.
